odoo/sql_db.py

```diff
@@ -356,15 +313,15 @@ def execute(self, query, params=None, log_exceptions=None):
             params = params or None
             res = self._obj.execute(query, params)
         except Exception as e:
-            if self._default_log_exceptions if log_exceptions is None else log_exceptions:
+            if log_exceptions:
                 _logger.error("bad query: %s\nERROR: %s", tools.ustr(self._obj.query or query), e)
             raise
 
+        delay = time.time() - start
```

Comment:
iz broken as it doesn't use `real_time` anymore, so rolled back to incorrect behaviour when freezegun is involved.

Reply:
Good point, I failed one rebase obviously.